packages/locale/lang/uz-uz.ts

```diff
@@ -63,7 +63,7 @@ export default {
       loading: 'Yuklanmoqda',
       noMatch: 'Mos maʼlumot yoʻq',
       noData: 'Maʼlumot yoʻq',
-      placeholder: 'Tanladizngiz',
+      placeholder: 'Tanladingiz',
```

Comment:
Perhaps the semantics of `Tanlang` are more consistent with，WDYT?

https://github.com/ElemeFE/element/blob/290e68ea6aa6c56b7d83182b650e3be4f77ff1b0/src/locale/lang/uz-UZ.js#L65

Reply:
Tanglang is more appropriate